doc/mf6io/mf6ivar/examples/gwe-est-example.dat

```diff
@@ -1,5 +1,7 @@
 BEGIN OPTIONS
-  LATENT_HEAT_VAPORIZATION
+  CPW   4180.0  
```

Comment:
Should we spell this out as HEAT_CAPACITY_WATER?

Reply:
Change implemented in 4358e1b

Reply:
Maybe I'm wrong but I don't think gwe-est-example.dat was updated with the new keywords.

Reply:
Updated with 33853d6